test/config_tests.erl

```diff
@@ -572,4 +694,46 @@ getmsg(Pid) ->
 
 
 n_handlers() ->
-    length(gen_event:which_handlers(config_event)).
+    length(handlers()).
+
+handlers() ->
+    Handlers = gen_event:which_handlers(config_event),
+    [Pid || {config_listener, {?MODULE, Pid}} <- Handlers].
+
+%% copy/paste from test_util
```

Comment:
Not in this repo. There's on going work to make this more independent from the rest of couch so that other projects can use it without requiring all of couch.

The real solution would be to move all of our test boilerplate code to the couch_tests repo and then make it a test dep though.